core/templates/pages/signup-page/signup-page.component.spec.ts

```diff
@@ -130,7 +130,7 @@ describe('Signup page', function() {
       expect(ctrl.emailPreferencesWarningText).toBe('');
     });
 
-    it('should send correct information if can receive email updates',
+    it('should send correct information when can receive email updates',
```

Comment:
Should successfully signup when user opts to receive email updates

Reply:
Done.